index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>JS Mastery Quiz</title>
+</head>
+<body>
+	<div style="margin:0,auto;width:100%;">
+		<form>
+			<h1>
+				Desired line height: <input type="number" name="txtLineHeight" id="txtLineHeight" value="12" style="width:100px;height:40px;font-size:20px;margin:50px;text-align:center"onkeypress="if (event.keyCode == 13) { 
+						document.getElementById('btnGrowTree').click();
+						}">
+				
+				Desired line character: <input type="text" name="txtLineChar" id="txtLineChar" value="*" maxlength="1" style="width:50px;height:40px;font-size:20px;margin:50px;text-align:center" onkeypress="if (event.keyCode == 13) { 
+						document.getElementById('btnGrowTree').click();
+						}">
+				
+				<button type="button" name="btnGrowTree" id="btnGrowTree" style="width:100px;height:100px;font-size:20px;margin:50px;text-align:center" onclick="tree({height: parseInt(document.getElementById('txtLineHeight').value), char: document.getElementById('txtLineChar').value});">
+					Grow Your Tree!
+				</button>
```

Comment:
All of the `on...` functionality you have in your markup should be moved to your javascript and attached to the DOM using event listeners. Inline JS is generally frowned upon.